torchtune/modules/attention.py

```diff
@@ -259,27 +259,30 @@ def forward(
             k = self.k_proj(y)
             v = self.v_proj(y)
 
+            # Apply positional embeddings
+            # k: [b, s_y, n_kv, h_d]
+            k = k.view(b, s_y, self.num_kv_heads, self.head_dim)
```

Comment:
```suggestion
            k = k.view(b, s_y, -1, self.head_dim)
```

Dont change the -1 part?

Reply:
Just curious what's the reason?

Reply:
@dvorjackz are you asking me why i suggested the change? If, that was just to preserve what was there before.

But if you are asking why this diff, then maybe summary explains